packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -102,55 +102,29 @@ export = {
       },
     },
   },
-  'Can export and import VirtualNode and perform actions'(test: Test) {
+  'Can import Virtual Nodes using ARN and attributes'(test: Test) {
     // GIVEN
     const stack = new cdk.Stack();
 
-    // WHEN
-    const mesh = new appmesh.Mesh(stack, 'mesh', {
-      meshName: 'test-mesh',
-    });
-
-    const node = mesh.addVirtualNode('test-node', {
-      dnsHostName: 'test.domain.local',
-      listener: {},
-    });
+    const meshName = 'testMesh';
+    const virtualNodeName = 'test-node';
+    const arn = `arn:aws:appmesh:us-east-1:123456789012:mesh/${meshName}/virtualNode/${virtualNodeName}`;
 
-    const stack2 = new cdk.Stack();
-
-    const node2 = appmesh.VirtualNode.fromVirtualNodeName(stack2, 'imported-node', mesh.meshName, node.virtualNodeName);
-
-    node2.addListeners({
-      portMapping: {
-        port: 8081,
-        protocol: appmesh.Protocol.TCP,
-      },
+    // WHEN
+    const virtualNode1 = appmesh.VirtualNode.fromVirtualNodeAttributes(stack, 'importedVirtualNode1', {
+      mesh: appmesh.Mesh.fromMeshName(stack, 'Mesh', meshName),
+      virtualNodeName: virtualNodeName,
     });
+    // THEN
+    test.equal(virtualNode1.mesh.meshName, meshName);
+    test.equal(virtualNode1.virtualNodeName, virtualNodeName);
 
+    // WHEN
+    const virtualNode2 = appmesh.VirtualNode.fromVirtualNodeArn(
+      stack, 'importedVirtualNode2', arn);
```

Comment:
Same comment here - leave the old test as-is, and create a new one for `fromVirtualNodeAttributes `.

Reply:
The old test here does not make much sense. It adds a listener to `node2` but never does any sort of validation on it. Adding the listener to the imported class does not actually make much sense, as it is modeled outside of the CDK and making a mutating call on it would not be valid.

I can revert it, but it does not provide much value.